ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/ros2/RDXROS2HeightMapVisualizer.java

```diff
@@ -232,6 +278,12 @@ public void getRenderables(Array<Renderable> renderables, Pool<Renderable> pool,
             heightMapGraphicNew.getRenderables(renderables, pool);
          }
 
+         if (enableGlobalHeightMapVisualizer.get())
+         {
+            globalHeightMapGraphic.getRenderables(renderables, pool);
+            heightMapGraphicNew.getRenderables(renderables, pool);
```

Comment:
Same for this, haven't tested it but if we are using an entirely different graphic I think we don't need to update the regular height map graphic here